### PR TITLE
Courses filter

### PR DESCRIPTION
**Pull request by ValentinaTenus**
added image upload
added default logo for courses ad companies
small changes in styles
changed filters logic for courses and companies

### CODE REVIEW

File: src/common/components/logo/index.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const Logo: React.FC<Properties> = ({ className, logo, name }) => {
+	const firstLetter = name
+		? name.charAt(FIRST_LETTER_INDEX).toUpperCase()
+		: "?";
+
+	return (
+		<>
+			{logo !== LOGO_NONE ? (
+				<img
+					alt={`${name} - logo`}
```
Comment (KirillSerg) [inline src/common/components/logo/index.tsx:25]: maybe just a name will be enough?

File: src/common/components/search-bar/index.tsx [hunk 2]
```diff
@@ -123,12 +119,17 @@ const SearchBar: React.FC<SearchBarProperties> = ({
 							</Button>
 						</div>
 					) : (
-						<IconButton className={styles["search__icon_button"]}>
```
Comment (KirillSerg) [inline src/common/components/search-bar/index.tsx:126]: I offer another option, just add optional props, like i did? but not pushet yet
image

**Review by KirillSerg — APPROVED** [review]

File: src/common/components/logo/index.tsx (hunk 1 above)
Comment (ValentinaTenus) [reply src/common/components/logo/index.tsx:25]: I think this name is more clear

File: src/common/components/search-bar/index.tsx (hunk 2 above)
Comment (ValentinaTenus) [reply src/common/components/search-bar/index.tsx:126]: which props do you mean?

**Pull request merged**